NEWS.md

```diff
@@ -116,8 +116,35 @@ differently [#3099].  There is a pending fix [wlroots-5159].
 
 [unreleased-commits]
 
+This release contains a good amount of bug-fixes, code simplification and
+small usability improvements.
+
+With the stability that comes with having tracked `wlroots 0.19` for a decent
+length of time, this feels like best version of labwc so far.
+
+In terms of new features, it is worth drawing attention to the click support in
+the window-switcher on-screen-display by @tokyo4j [#3186] which has frequently
+been requested by users.
+
+As a general note to users, we discourage the use of empty strings in the
+`rc.xml` configuration file, for example `<margin output=""/>`. There are only a
+few areas left where empty string are ignored (like under `<libinput>`) but the
+intent for future releases is to consistently read empty strings as empty
```

Comment:
I may have missed some discussion on this point, but I wonder if this "intent for future releases" is a good idea. I personally like the new "ignore and warn" behavior that got merged for 0.9.3. Offhand, I can't think of a good reason to explicitly use an empty string, so to "read empty strings as empty strings" (i.e. to give a different meaning to an empty string vs. an omitted string) seems like it would just cause confusion. The warnings seems like a good hint: "Hey, this probably doesn't do what you intended."

Reply:
There are some cases like the workspace prefix or the custom window switcher field / thumbnail format strings. If these are set to an empty string they would mean an empty string. In contrast, if absent they would be using the default values.